tests/e2e/e2e_common.go

```diff
@@ -105,7 +107,7 @@ const (
 	quotaName                                  = "cns-test-quota"
 	regionKey                                  = "failure-domain.beta.kubernetes.io/region"
 	resizePollInterval                         = 2 * time.Second
-	rqLimit                                    = "200Gi"
+	rqLimit                                    = "900Gi"
```

Comment:
don't change this

Reply:
done, I have created a new constant for the same